changelog/change_preserve_multiline_semantics_on_rails_presence.md

```diff
@@ -0,0 +1 @@
+* [#817](https://github.com/rubocop/rubocop-rails/pull/817): Preserve multiline semantics on `Rails/Presence`. ([@ydah][])
```

Comment:
Can you remove this entry? Because not required for message tweaking.

Reply:
I remove this entry. Thank you so much!